### DISCUSSION TITLE
url управления устройством

### DISCUSSION THREAD

**Discussion by IljaGolenev** [GyverLibs/GyverPortal#84] [Q&A]
Вот написал я, допустим, вебинт. с одной кнопкой чтоб модулем реле управлять, вывел esp в мировой интернет и имею доступ из любой сети.
Как узнать url адреса элементов, что бы можно было в навыке домовенок кузя (для Алисы), подключить esp используя правило get?
![image](https://user-images.githubusercontent.com/88191098/235382811-53901642-f1e2-45d7-8fd2-a341181d3170.png)

**Comment by GyverLibs** [comment]
Для установки значения (событие клик) отправляется запрос вида:
http://192.168.1.154/GP_click?имя=значение
А вообще проброс url это костыль, лучше использовать mqtt

**Comment by GdalexAlex** [comment]
Добрый день!
А есть ли какой-нибудь метод (способ) вызвать событие FORM_CLICK или UPDATE из кода? Например, при нажатии на кнопку на форме код отрабатывает какое-то действие и после выполнения нужно страницу обновить. Как это можно сделать?

**Reply by GdalexAlex** [reply]
Обошёл это ограничение путём добавления JS, который каждую секунду анализирует содержимое <outer>, который обновляется из кода по событию UPDATE. Когда он получает нужное значение, перегружаю страницу из самого HTML методом location.reload();
